src/store/index.js

```diff
@@ -126,7 +128,10 @@ export default new Vuex.Store({
 		},
 		SHOW_FEEDBACK_WIDGET(state, choice) {
 			this.state.active_user.onboarding.likehihoodToShare = choice
-		}
+		},
+		setRedirectPath(state, path) {
```

Comment:
This is an action right, it does not belong here under mutations. Leave move it to actions object. Also use snake_case for consistency